src/sentry/api/serializers/models/organization.py

```diff
@@ -121,6 +122,22 @@ def serialize(self, obj, attrs, user):
                 default=100,
             )),
         }
+        context.update({
+            'isDefault': obj.is_default,
+            'defaultRole': obj.default_role,
+            'availableRoles': [{
+                'id': r.id,
+                'name': r.name,
+            } for r in roles.get_all()],
+            'openMembership': bool(obj.flags.allow_joinleave),
+            'allowSharedIssues': not obj.flags.disable_shared_issues,
```

Comment:
mostly i was just sticking with the meaning of things, and the reason the field is disable vs enable is because you cant reasonably set defaults on a bitfield